arch/Kconfig

```diff
@@ -26,6 +26,7 @@ config ARM
 	bool "ARM architecture"
 	select ARCH_HAS_THREAD_ABORT
 	select HAS_DTS
+	select ARCH_HAS_NOCACHE_MEMORY_SUPPORT if ARM_MPU && CPU_CORTEX_M7
```

Comment:
Why is this selection needed, if the symbol is selected in CPU_CORTEX_M7?

Reply:
That's a leftover from previous version, I'll fix it in the next version.